src/components/dateline.stories.tsx

```diff
@@ -2,15 +2,18 @@
 
 import React, { FC } from 'react';
 import { date, withKnobs } from '@storybook/addon-knobs';
-
 import { some } from '@guardian/types/option';
 import Dateline from './dateline';
+import { Pillar } from '@guardian/types/Format';
 
+type DefaultProps = {
+    pillar: Pillar
+}
 
 // ----- Stories ----- //
 
-const Default: FC = () =>
-    <Dateline date={some(new Date(date('Publish Date', new Date('2019-12-17T03:24:00'))))} />
+const Default: FC<DefaultProps> = ({pillar}:DefaultProps) =>
```

Comment:
I think you can pass pillar directly into Dateline like they pass the props in here? 

https://github.com/guardian/apps-rendering/blob/a35bbe2cc861099b157888a2692e2ecc8a81f665/src/components/anchor.stories.tsx#L19

Reply:
Trying to replica the model with little success:
const Default: FC = () =>
    <Dateline 
        pillar={{
            pillar: Pillar
        }}

Reply:
I think what you have now is ok, you just might not need type and argument to the default component?

<Dateline pillar={pillar} date={some(new Date(date('Publish Date', new Date('2019-12-17T03:24:00'))))} />

Reply:
coming up with error: Cannot find name pillar when i do that, thats why i initially used the prop DefaultProps to feed it in.